.github/workflows/chromatic-pr.yaml

```
@@ -0,0 +1,74 @@
+# Chromatic action from https://www.chromatic.com/docs/github-actions
+
+name: "chromatic and storybook tests (PR)"
```

```suggestion
name: "test storybook"
```

Should cover both cases

---

Can do. I'll rename the file too.